.github/workflows/continuous-monitoring.yml

```diff
@@ -0,0 +1,39 @@
+name: Continuous monitoring of distribution channels
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */1 * * *'
+
+jobs:
+  smoke-tests:
+    name: Run smoke tests for Node ${{ matrix.node-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
```

Comment:
think we can just keep `ubuntu-latest` and latest version of node should be good enough.

Reply:
Just updated